### PR TITLE
For BZ1643169 -- configuring storage Option A dynamic provisioning -- non-NFS

### PR DESCRIPTION
**Pull request by jhoyt-rh**
@gkimetto@redhat.com, PTAL
@vikram-redhat PTAL
@openshift/team-documentation PTAL

Doc reviewers, I wanted to get some initial review on this, especially for the xref formatting, etc., as well as clarity of content. And is there a better/preferred way to make the references?

Based on master, and will be CPed to 3.10 and 3.11.

### CODE REVIEW

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -1711,16 +1711,23 @@ installation:
 [[advanced-installation-logging-storage-dynamic]]
 ==== Option A: Dynamic
 
-If your {product-title} environment supports
-xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamic volume provisioning]
-for your cloud provider, use the following variable:
+If your {product-title} environment has dynamic volume provisioning, it could be configured
+either via the cloud provider or by an independent storage provider. For instance, the cloud provider
+could have a StorageClass with provisioner `kubernetes.io/gce-pd` on GCE, and an
+independent storage provider such as glusterfs could have a StorageClass with provisioner
```
Comment (vikram-redhat) [inline install/configuring_inventory_file.adoc]: glusterfs should be GlusterFS.

**Comment by wongma7** [comment]
lgtm on my side :+1:

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (kimettog) [reply install/configuring_inventory_file.adoc]: Looks good to me

**Review by kimettog — COMMENTED** [review]
Looks good to me

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (jhoyt-rh) [reply install/configuring_inventory_file.adoc]: THX. Incorporated those 2 edits.

**Pull request merged**